modules/avformat/avformat.c

```diff
@@ -303,7 +303,7 @@ int avformat_shared_alloc(struct shared **shp, const char *dev,
 		}
 	}
 
-	if (strnlen(avformat_inputformat, sizeof avformat_inputformat)	> 0) {
+	if (str_isset(avformat_inputformat)	> 0) {
```

Comment:
`str_isset` returns bool. the code should be:

```c
if (str_isset(avformat_inputformat))
```

Reply:
ooops